src/cloud/live/sens-data-over.md

```diff
@@ -13,7 +13,7 @@ You must have [environment-level, Admin role privileges][admin] to complete conf
 
 ## Consistent configuration
 
-The database contains default configurations for your Magento store. When you update configurations in the {{site.data.var.ece}} environments using the Magento Admin panel, your configuration changes apply to the `app/etc/config.php` file. You can then use this file to manage and synchronize the Magento application configuration for your local environment and across each Cloud environment.
+The database contains default configurations for your Magento store. When you update configurations in the {{site.data.var.ece}} environments using the Magento Admin panel or the Magento CLI command  `bin/magento config:set`, your configuration changes apply to the `app/etc/config.php` file. You can then use this file to manage and synchronize the Magento application configuration for your local environment and across each Cloud environment.
```

Comment:
```
When you update configurations in the {{site.data.var.ece}} environments using the Magento Admin panel or the Magento CLI command  `bin/magento config:set`, your configuration changes apply to the `app/etc/config.php` file
```
If customer changes configurations in Magento Admin panel - these changes do not affect/apply in `app/etc/config.php`. The same for  `bin/magento config:set` all configurations are saved to DB by default. Only if you use option `--lock-config` then values of configuration will be saved to `app/etc/config.php`.